goserver/server/server.go

```diff
@@ -38,9 +50,18 @@ func (s *Server) handleLaunch(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "missing iss in URL query parameters", http.StatusUnauthorized)
 		return
 	}
-	if iss[0] != s.AuthorizedFHIRURL {
+	if iss[0] != s.authorizedFHIRURL {
 		http.Error(w, fmt.Sprintf("unauthorized iss %s", iss[0]), http.StatusUnauthorized)
 		return
 	}
+
+	sess, err := s.sm.New(w, r)
+	if err != nil {
+		http.Error(w, "cannot create session", http.StatusInternalServerError)
+		return
+	}
+	sess.FHIRURL = iss[0]
+	s.sm.Save(sess)
```

Comment:
Check for error and add unit test covering this.

Reply:
I see. It's using httptest.NewServer with a passed in handler. (I tried to use http.Server and my Mac asked me for permission everytime I ran the test)

Reply:
error check added.
Test will be added in a following PR since it will change this handler.

Reply:
What do you mean it will change the handler? I'm usually not a big fan of leaving unit tests for follow up CLs as they tend to get forgotten quite easily.

Reply:
Sorry, it turns out the handler will only be added with new logic but the old logic will not be changed - tests added.